test/ClearValue.lean

```diff
@@ -0,0 +1,21 @@
+import Mathlib.Tactic.ClearValue
+
+example {a b : Nat} (h : a = b) : a = b := by
```

Comment:
There should be a test including dependent types, where clearing the value would result in an ill-typed goal.